backend/models/response_user.py

```diff
@@ -0,0 +1,16 @@
+from backend.models.user import User
+
+
+# This class should be used for sending users information to client.
+# We don't want to send sensitive information such as password or email.
+class ResponseUser:
+    def __init__(self, user: User):
+        self.id = user.id
+        self.username = user.username
+        self.first_name = user.first_name
+        self.last_name = user.last_name
+        self.email = user.email
```

Comment:
Is email used as part of this class?

Reply:
Yes, and I just now noticed that we should change the description above the class declaration to "such as password" instead of "such as password or email". Will be fixed.

Reply:
We have deleted this file just now, so my comment not relevant